content/en/docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -559,8 +559,8 @@ You can view application logs written to standard out or standard error using `k
 ```
 
 Kubernetes supports more powerful, but more complex, logging integrations
-with [Stackdriver](/docs/tasks/debug-application-cluster/logging-stackdriver/)
-and [Elasticsearch and Kibana](/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/).
+with [Stackdriver](https://cloud.google.com/stackdriver/docs)
+and [Elasticsearch and Kibana](https://www.elastic.co/elastic-stack).
```

Comment:
I think this is dated. By now, several more products work with Kubernetes for logging. It's a long list.

We should tell people that Kubernetes leaves it up to the cluster operator to select a logging solution and integrate it with their cluster or application.

Reply:
Sure. I can update this.